### PR TITLE
CIRC-5807 Show how to batch timestamped metrics

### PR DESCRIPTION
**Pull request by esproul**
Change URL in example to use the current best practice of the API
host, which removes the need for a custom CA cert.

Update Advanced Configuration section.

### CODE REVIEW

File: content/circonus/checks/check-types/httptrap.md [hunk 1]
```diff
@@ -37,11 +34,15 @@ This will allow you to select and deselect metrics you want to collect. Click th
 
 ## Advanced Configuration
 
-The "Period" refers to how often the check runs. If asynchronous collection is enabled, each value will be remembered as soon as it's received. Then once per period, the values from that period will be averaged and the average will be stored. Otherwise, only the most recent value will be stored, once per period.
+The "Period" refers to how often values are accumulated and stored. If
+asynchronous collection is enabled (default), each value will be remembered as
+soon as it's received. Then once per period, the values from that period will
+be averaged and the average will be stored. Otherwise, only the most recent
+value will be stored, once per period.
```
Comment (bdunavant) [inline content/circonus/checks/check-types/httptrap.md:41]: Discussion with Eric.  I don't think async batches things by period.  We'll have to test.

File: content/circonus/checks/check-types/httptrap.md [hunk 2]
```diff
@@ -97,12 +98,29 @@ Numeric values for histograms can be provided in two additional ways:
...
  * As a prebucketed histogram. For example `["H[0.1]=3", "H[11]=7"]`, would mean that in the bin 0.1 (which is 0.10 to 0.11) there are 3 samples and in the bin 11 (which is 11 to 12) there are 7 samples.
 
-If HTTPtrap submissions contain an extended value with `_ts`, the individual measurement will be timestamped with the provided valid instead of the default "now."  The value of `_ts` should be specified in millisecond since UNIX epoch 1970-01-01 00:00:00-0000.  `_ts` is a peer to the `_type` and `_value` keys specified above.
+### Timestamped Submission
+
+If HTTPtrap submissions contain an additional value with `_ts`, the individual
+measurement will be timestamped with the provided value instead of the default
+"now."  The value of `_ts` should be specified in milliseconds since UNIX epoch
+1970-01-01 00:00:00-0000.  `_ts` is a peer to the `_type` and `_value` keys
+specified above.
```
Comment (bdunavant) [inline content/circonus/checks/check-types/httptrap.md:107]: Fairly certain that providing a _ts value causes it to no longer be applicable to the batching in `period` above.  We should verify and probably include something here about that.

File: content/circonus/checks/check-types/httptrap.md [hunk 3]
```diff
@@ -97,12 +98,29 @@ Numeric values for histograms can be provided in two additional ways:
...
+
+Multiple measurements may be batched into a single JSON document, as well:
+```
+{
+    "foo": { "_type": "n", "_value": 1, "_ts": 1604936367789 },
+    "foo": { "_type": "n", "_value": 2, "_ts": 1604936552774 },
+    "foo": { "_type": "n", "_value": 3, "_ts": 1604936616422 }
+}
+```
+
```
Comment (bdunavant) [inline content/circonus/checks/check-types/httptrap.md:117]: Should we also include an example of streaming json?   That's actually a lot easier for people to implement anyway IMO

Comment (esproul) [reply content/circonus/checks/check-types/httptrap.md:117]: I don't have a good test for that; I am not sure that sending curl a string comprised of several complete json documents actually works or not.

File: content/circonus/checks/check-types/httptrap.md (hunk 1 above)
Comment (esproul) [reply content/circonus/checks/check-types/httptrap.md]: Clarified with further discussion and testing.

File: content/circonus/checks/check-types/httptrap.md (hunk 2 above)
Comment (esproul) [reply content/circonus/checks/check-types/httptrap.md:111]: Added a statement to this effect.

File: content/circonus/checks/check-types/httptrap.md (hunk 3 above)
Comment (esproul) [reply content/circonus/checks/check-types/httptrap.md:127]: Validated that curl can do multiple documents in one data string. Included an example of that.

**Review by bdunavant — APPROVED** [review]
LGTM

**Pull request merged**